sections/earn/LPTab/DepositTab/DepositTab.tsx

```diff
@@ -106,26 +102,21 @@ const DepositTab: FC<DepositTabProps> = ({
 			? blockExplorerInstance.txLink(txHash)
 			: undefined;
 
-	const stakedBalanceDisplay = staked * pricePerShare;
+	const stakedBalanceDisplay = staked.mul(pricePerShare);
+
+	let parsedAmount = wei(0);
+	try {
+		parsedAmount = wei(amount);
+	} catch {}
```

Comment:
shouldn't we do something in the catch block?

Reply:
probably don't need a ```try catch``` here, if we move ```wei(amount``` to L118 the ```try, catch``` will catch any errors there in the ```getGasLimitEstimate```